x-pack/plugins/security_solution/server/lib/detection_engine/routes/rules/utils/deprecation.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { getDocLinks } from '@kbn/doc-links';
+import { Logger } from 'src/core/server';
+import { DETECTION_ENGINE_RULES_BULK_ACTION } from '../../../../../../common/constants';
+
+/**
+ * Helper method for building deprecation messages
+ *
+ * @param path Deprecated endpoint path
+ * @returns string
+ */
+export const buildDeprecatedBulkEndpointMessage = (path: string) => {
+  const docsLink = getDocLinks({ kibanaBranch: 'main' }).siem.ruleApiOverview;
```

Comment:
wonder, is it might be better to give a link to a latest stable release, rather main branch?